DOC-828 Add exists operator to cache processor (#123)
redpanda-data/rp-connect-docs: modules/components/pages/processors/cache.adoc

with the result. If the key does not exist the action fails with an error, which
can be detected with xref:configuration:error_handling.adoc[processor error handling].

=== `exists`

> Check whether a specific key is already cached and replace the original message payload with `true` if the key exists in the cache, or `false` if it doesn't.

Reviewer: ```suggestion
Check if a specific key is in the cache and set the message payload to `true` if it exists, or `false` if it doesn’t.
```

Suggest this simplification.

Author: Updated but kept consistent with other descriptions.
